recipes/sdl/all/conanfile.py

```diff
@@ -150,6 +150,10 @@ def validate(self):
         if self.settings.os == "Macos" and not self.options.iconv:
             raise ConanInvalidConfiguration("On macOS iconv can't be disabled")
 
+        # SDL>=2.0.18 requires xcode 12 or higher because it uses CoreHaptics.
+        if tools.Version(self.version) >= "2.0.18" and tools.is_apple_os(self.settings.os) and tools.Version(self.settings.compiler.version) < "12":
```

Comment:
that's really weird, CoreGraphics was available very long before Xcode 12...

Reply:
It's not CoreGraphis. 
CoreHaptics is introduced to support haptic feedback of game controller.
Please refer official document. https://developer.apple.com/documentation/corehaptics/playing_haptics_on_game_controllers

Reply:
It's [CoreHaptics](https://developer.apple.com/documentation/corehaptics). According the link it says +13, but I have no idea.

Reply:
oh, yes, I was reading too fast xD